mmv1/third_party/terraform/services/container/resource_container_node_pool_test.go.tmpl

```diff
@@ -550,7 +564,7 @@ func TestAccContainerNodePool_withKubeletConfig(t *testing.T) {
 				ImportStateVerify: true,
 			},
 			{
-				Config: testAccContainerNodePool_withKubeletConfig(cluster, np, "", "", networkName, subnetworkName, "FALSE", false, 1024),
+				Config: testAccContainerNodePool_withKubeletConfig(cluster, np, "", "", networkName, subnetworkName, "FALSE", "500Mi", "30s", "", false, 1024, 5, 50, 80),
```

Comment:
The test is failing with `"Invalid kubelet container_log_max_size and container_log_max_files: the total container log size 2.44Gi cannot exceed 1% of the total storage 100Gi."` Reducing `500Mi` to `200Mi` should get you past this error while still testing updating the field.